### DISCUSSION TITLE
new ai

### DISCUSSION THREAD

**Discussion by NONLOSAINEMMENOTU** [community/community#80784] [Programming Help]
Etica dell'IA: La questione etica è fondamentale quando si tratta di IA. Si discute spesso su come garantire che l'IA sia utilizzata in modo responsabile, evitando discriminazioni, assicurando la trasparenza e affrontando le questioni legate alla privacy.

**Comment by NONLOSAINEMMENOTU** [comment]
ai discussion

**Comment by ghostinhershell** [comment]
Hi @Laygandolfo06, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
